node/resolvers/appsWithStats.ts

```diff
@@ -1,14 +1,16 @@
-import { removeVersionFromAppId } from '@vtex/api'
-import { compose, filter, isEmpty, isNil, keys, map, reject, startsWith } from 'ramda'
+import { filter, includes, map, pluck, prop } from 'ramda'
 
-const nullable = x => isEmpty(x) || isNil(x)
 
 export const appsWithStats = async (root, args, ctx: Context, info) => {
-  const {dataSources: {apps}, vtex: {account}} = ctx
-  const deps = await apps.getDependencies().then(keys)
-  return compose(
-    reject(nullable),
-    map(removeVersionFromAppId),
-    filter(startsWith(account))
-  )(deps)
-}
\ No newline at end of file
+  const {dataSources: {registry}, vtex: {account}} = ctx
+  const publishedAppsInThisAccount = await registry.listApps()
+    .then(prop('data'))
+    .then(pluck('partialIdentifier'))
+    .then(result => filter(includes(account), result))
```

Comment:
It is unlikely but what if the string account is in an app's name?
Example: `vendor.appvtex` can be interpreted as an app from vendor vtex.

Reply:
makes sense